source/read/cursors.txt

```diff
@@ -0,0 +1,104 @@
+.. _java-rs-cursors:
+
+=========================
+Access Data From a Cursor
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: read, results, oplog
+
+Overview
+--------
+
+In this guide, you can learn how to access data from a **cursor** by using the
```

Comment:
This is an interesting one. With reactive streams, cursors really are an implementation detail of the stream and not really a user facing detail to be documented.

I think we either add a big caveat that this is the case, eg. 

In the mongodb reactive stream driver, some streams are backed by cursors. The size of batches used in these underlying cursors depend on the demand requested on the Subscription for the Publisher.

Reply:
I'll include an addendum like this to the intro!